schemas/Clinical/index.js

```diff
@@ -178,6 +178,11 @@ const typeDefs = gql`
     Retrieve current stored Clinical Submission Data Dictionary Schema version
     """
     clinicalSubmissionSchemaVersion: String!
+
+    """
+    Retrieve current Clinical Submission System disabled state
+    """
+    clinicalSubmissionSystemState: Boolean!
```

Comment:
rename to something like `clinicalSubmissionSystemDisabled` or `clinicalSubmissionSystemEnabled`